language/stdlib/transaction_scripts/doc/overview.md

```diff
@@ -1,121 +1,4001 @@
+
+
+---
+
+<a name="@Script_unfreeze_account_40"></a>
+
+#### Script unfreeze_account
+
+
+Unfreezes the account at <code>address</code>. The sending account of this transaction must be the
+Treasury Compliance account. After the successful execution of this transaction transactions
+may be sent from the previously frozen account, and coins may be sent and received.
+
+Script documentation: <code><a href="overview.md#unfreeze_account">unfreeze_account</a></code>
+
+
+---
+
+<a name="@Script_update_dual_attestation_limit_41"></a>
+
+#### Script update_dual_attestation_limit
+
+
+Update the dual attestation limit on-chain. Defined in terms of micro-LBR.  The transaction can
+only be sent by the Treasury Compliance account.  After this transaction all inter-VASP
+payments over this limit must be checked for dual attestation.
+
+Script documentation: <code><a href="overview.md#update_dual_attestation_limit">update_dual_attestation_limit</a></code>
+
+
+---
+
+<a name="@Script_update_exchange_rate_42"></a>
+
+#### Script update_exchange_rate
+
+
+Update the rough on-chain exchange rate between a specified currency and LBR (as a conversion
+to micro-LBR). The transaction can only be sent by the Treasury Compliance account. After this
+transaction the updated exchange rate will be used for normalization of gas prices, and for
+dual attestation checking.
+
+Script documentation: <code><a href="overview.md#update_exchange_rate">update_exchange_rate</a></code>
+
+
+---
+
+<a name="@Script_update_minting_ability_43"></a>
+
+#### Script update_minting_ability
+
+
+Script to allow or disallow minting of new coins in a specified currency.  This transaction can
+only be sent by the Treasury Compliance account.  Turning minting off for a currency will have
+no effect on coins already in circulation, and coins may still be removed from the system.
+
+Script documentation: <code><a href="overview.md#update_minting_ability">update_minting_ability</a></code>
+
+
+
+<a name="@System_Administration_44"></a>
+
+### System Administration
+
+
+---
+
+<a name="@Script_update_libra_version_45"></a>
+
+#### Script update_libra_version
+
+
+Updates the Libra major version that is stored on-chain and is used by the VM.  This
+transaction can only be sent from the Libra Root account.
+
+Script documentation: <code><a href="overview.md#update_libra_version">update_libra_version</a></code>
+
+
+---
+
+<a name="@Script_add_to_script_allow_list_46"></a>
+
+#### Script add_to_script_allow_list
+
+
+Adds a script hash to the transaction allowlist. This transaction
+can only be sent by the Libra Root account. Scripts with this hash can be
+sent afterward the successful execution of this script.
+
+Script documentation: <code><a href="overview.md#add_to_script_allow_list">add_to_script_allow_list</a></code>
+
+
+
+---
+
+<a name="@Transaction_Scripts_47"></a>
+
+## Transaction Scripts
+
+---
+
+The set of transaction scripts that are allowed to be sent to the blockchain
```

Comment:
Duplicate of this sentence + list from above?

Reply:
Good spot. I'll update